emscripten.py

```diff
@@ -2109,6 +2139,8 @@ def create_s2wasm_args(temp_s):
   args += ['--global-base=%d' % shared.Settings.GLOBAL_BASE]
   args += ['--initial-memory=%d' % shared.Settings.TOTAL_MEMORY]
   args += ['--allow-memory-growth'] if shared.Settings.ALLOW_MEMORY_GROWTH else []
+  args += ['--emscripten-reserved-function-pointers=%d' %
```

Comment:
As a direct contrast to what I said for `wasm-emscripten-finalize`, because this is the only element between square brackets it doesn't need parentheses or extra indentation.

Reply:
You mean the way it is looks ok, right?

Reply:
Yes, was (somewhat clumsily) pointing out that this looks good as-is